xbmc/guilib/WindowIDs.h

```diff
@@ -169,7 +169,14 @@
 #define WINDOW_VISUALISATION              12006
 #define WINDOW_SLIDESHOW                  12007
 #define WINDOW_DIALOG_COLOR_PICKER 12008
-#define WINDOW_DIALOG_VIDEO_VERSION_SELECT 12009
+
+// Note: Numbers given here must match the ids given in strings.po for a translatable string for
```

Comment:
This should probably be a todo right? I.e. gluing windows and respective translations together should be avoided. At least I am doing this for the picture stuff

Reply:
Yes, will make this a todo.